lib/perl/Genome/Model/Tools/Bsmap/methylation_ratio.py

```diff
@@ -0,0 +1,181 @@
+'''
+Created on Jan 26, 2015
+
+@author: ygindin
+'''
+import optparse
+import logging
+import time
+import tempfile
+import pysam
+import os
+import pdb
+from multiprocessing import Process, Queue
+
+def write_data(result_queue, out_file, work_units):
+    start = time.clock()
+    fout = open(out_file, 'w')
+    fout.write('chr\tpos\tratio\ttotal_C\tmethy_C\tunmeth_C\n')
+    tab = '\t'
+    new_line = '\n'
+    print 'writer opens shop. waiting for ', work_units, ' work units'
```

Comment:
I guess this applies throughout the script.  I guess there's no harm in printing to stdout since you are writing to a file.  However, it is often the case that the default output (when no file specified) would be stdout in which case these `print` calls would interfere.